mkdocs.yml

```diff
@@ -52,6 +52,7 @@ nav:
       - Community:
           - "How to contribute": Contributing.md
           - "Communication Channels": Communication-Channels.md
+          - "Governance": Governance.md
```

Comment:
The Community submenu is no longer there now we merged everything in the Contributing page. I would move this page one level up, e.g. before or after Terms.

Reply:
:+1: